main.go

```diff
@@ -102,6 +108,8 @@ type Service struct {
 	Privileged  bool
 	Command     string
 	Environment map[string]string
+        Log_Driver   string
+        Log_Opt      map[string]string
```

Comment:
Could just remove the underscore? Golang pattern does not use underscore for variable names. Also the indentation for the new variables are different from those above.